tools/java/common/pom.xml

```diff
@@ -49,6 +49,12 @@
     </plugins>
   </build>
 
+  <repositories>
+    <repository>
+      <id>protobuf-java-lib</id>
+      <url>file://${project.basedir}/../../../java/lib</url>
+    </repository>
+  </repositories>
   <dependencies>
     <dependency>
       <groupId>junit</groupId>
```

Comment:
Fun fact: we have junit-4.8.1.jar under java/lib. I wonder what Maven is using, and who is using the one under java/lib.